Makefile

```diff
@@ -19,11 +19,7 @@ TEST_FLAGS ?= -v # -race
 
 .PHONY: falcoctl
 falcoctl:
-	$(GO) build -ldflags \
-    "-X ${PROJECT}/pkg/version.semVersion=${RELEASE} \
-    -X ${PROJECT}/pkg/version.gitCommit=${COMMIT} \
-    -X ${PROJECT}/pkg/version.buildDate=${BUILD_DATE}" \
-    -o falcoctl .
+	$(GOREL) build --rm-dist --single-target --snapshot
```

Comment:
Although I love to use `goreleaser`, I'd not force developers to install it for local builds.

Could we use two distinct targets so that `make all` would not require `goreleaser`?